disnake/embeds.py

```diff
@@ -794,3 +797,66 @@ def _handle_resource(self, url: Optional[Any], file: File, *, key: _FileKey) ->
         else:
             self._files.pop(key, None)
             return str(url) if url is not None else None
+
+    def check_limits(self):
```

Comment:
A reference to this method in the `len` docs would be nice:
https://github.com/DisnakeDev/disnake/blob/ccac417d03bf5e571263ba09bf67402c021a47c7/disnake/embeds.py#L122-L127